.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,21 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    echo "=== Injecting region tags into XUnit output ==="
+    XUNIT_PATH="$PWD/sponge_log.xml"
+    XUNIT_TMP_PATH="tmp/drift_tmp.xml"
+    if [[ -f "$XUNIT_PATH" ]]; then
+        echo "Processing XUnit output file: $XUNIT_PATH"
+        cat "$XUNIT_PATH" | python3.7 "$PARSER_PATH" inject-snippet-mapping "$PWD" > "$XUNIT_TMP_PATH"
+        mv $XUNIT_TMP_PATH $XUNIT_PATH
```

Comment:
I think you have to check the return value, maybe

```
if [[ $? -eq 0 ]]; then
    mv $XUNIT_TMP_PATH $XUNIT_PATH
fi
```
?

Reply:
Not necessary if you expect the script to consistently succeed. 😉 

(I added a guard here anyways, as it makes failures easier to debug.)

Reply:
Well, it's safe to assume that any process can fail for random reasons.